File: 1662.check-if-two-string-arrays-are-equivalent.java
/*
 * @lc app=leetcode id=1662 lang=java
 *
 * [1662] Check If Two String Arrays are Equivalent
 */

// @lc code=start
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
        
        String string1 = "";
        String string2 = "";

        for(String str : word1) string1 += str;
        for(String str : word2) string2 += str;

        if(string1.equals(string2)) return true;

        return false;
        
    }
}
// @lc code=end

File: 1662.check-if-two-string-arrays-are-equivalent.js
/*
 * @lc app=leetcode id=1662 lang=javascript
 *
 * [1662] Check If Two String Arrays are Equivalent
 */

// @lc code=start
/**
 * @param {string[]} word1
 * @param {string[]} word2
 * @return {boolean}
 */
var arrayStringsAreEqual = function(word1, word2) {
    
    let string1 = word1.join('');
    let string2 = word2.join('');

    if(string1 === string2) return true;

    return false;
};
// @lc code=end

